tabular/src/autogluon/tabular/models/tabular_nn/tabular_nn_trial.py

```diff
@@ -26,3 +26,23 @@ def tabular_nn_trial(args, reporter):
         if not isinstance(e, TimeLimitExceeded):
             logger.exception(e, exc_info=True)
         reporter.terminate()
+
+
+@args()
+def tabular_pytorch_trial(args, reporter):
```

Comment:
Put in separate file, so TabularPyTorchDataset is not imported unless needed.

Reply:
moving to new file `tabuler_nn_torch.py`